lib/lz4.h

```diff
@@ -546,9 +546,9 @@ LZ4_decompress_safe_partial_usingDict(const char* src, char* dst,
 #define LZ4_STATIC_3504398509
 
 #ifdef LZ4_PUBLISH_STATIC_FUNCTIONS
-#define LZ4LIB_STATIC_API LZ4LIB_API
+# define LZ4LIB_STATIC_API LZ4LIB_API
```

Comment:
How does this work with the space after the `#`?

Reply:
https://gcc.gnu.org/onlinedocs/cpp/The-preprocessing-language.html

> 1.4 The preprocessing language
> ... Whitespace is allowed before and after the '`#`' ...

See also C11 draft - 6.10 Preprocessing directives
https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf